gen/_template/server.tmpl

```diff
@@ -4,10 +4,12 @@
 {{ template "header" $ }}
 
 {{- if $.PathsServerEnabled }}
-{{- with $ops := $.Operations }}
 // Handler handles operations described by OpenAPI v3 specification.
 type Handler interface {
-{{- range $op := $ops }}
+{{- range $group := $.OperationGroups }}
+	{{ $group.Name }}Handler
```

Comment:
Probably [`Invoker`](https://github.com/ogen-go/ogen/blob/e1628ac9f35fb4ea64e574bb089035b28fecfa62/gen/_template/client.tmpl#L8-L19) interface should be split into groups too.

Reply:
Thanks. I've made the change to split up the `Invoker` interface as well.